Sources/Shared/Request+Album.swift

```diff
@@ -29,13 +29,71 @@ public typealias AlbumRequest = Request<Album>
 /// `Request` that returns an array of `Album` objects.
 public typealias AlbumListRequest = Request<[Album]>
 
+fileprivate enum Key {
+    static let name = "name"
+    static let description = "description"
+    static let password = "password"
+    static let privacy = "privacy"
+}
+
 public extension Request {
-    
     /// Returns a new request to fetch a specific album.
     ///
     /// - Parameter uri: The album's URI.
     /// - Returns: Returns a new `Request` for an individual album.
     static func albumRequest(for uri: String) -> Request {
```

Comment:
The naming here seems ambiguous, is this a `fetchAlbum` request? 
Also do we always append request to the method names in a Request type? Seems redundant from the calling site i.e. `Request.createAlbum(...)` instead of `Request.createAlbumRequest(...)`

Reply:
I agree that the naming is weird and could be improved, but it's consistent with how (most) of our other request methods are named right now.